packages/@aws-cdk/aws-dynamodb/test/test.dynamodb.ts

```diff
@@ -229,8 +229,165 @@ export = {
+              MyTableReadAutoScalingRoleFEE68E49:
+               { Type: 'AWS::IAM::Role',
+                 Properties:
+                  { AssumeRolePolicyDocument:
+                     { Statement:
+                        [ { Action: 'sts:AssumeRole',
+                            Effect: 'Allow',
+                            Principal: { Service: 'application-autoscaling.amazonaws.com' } } ],
+                       Version: '2012-10-17' } } },
+              MyTableReadAutoScalingRoleDefaultPolicyF6A1975F:
+               { Type: 'AWS::IAM::Policy',
+                 Properties:
+                  { PolicyDocument:
+                     { Statement:
+                        [ { Action: [ 'dynamodb:DescribeTable', 'dynamodb:UpdateTable' ],
+                            Effect: 'Allow',
+                            Resource: { 'Fn::GetAtt': [ 'MyTable794EDED1', 'Arn' ] } },
+                          { Action: 'cloudwatch:*', Effect: 'Allow', Resource: '*' } ],
+                       Version: '2012-10-17' },
+                    PolicyName: 'MyTableReadAutoScalingRoleDefaultPolicyF6A1975F',
+                    Roles: [ { Ref: 'MyTableReadAutoScalingRoleFEE68E49' } ] } },
+              MyTableReadCapacityScalableTarget72B0B3BF:
+               { Type: 'AWS::ApplicationAutoScaling::ScalableTarget',
+                 Properties:
+                  { MaxCapacity: 500,
+                    MinCapacity: 50,
+                    ResourceId:
+                     { 'Fn::Join': [ '', [ 'table/', { Ref: 'MyTable794EDED1' } ] ] },
+                    RoleARN:
+                     { 'Fn::GetAtt': [ 'MyTableReadAutoScalingRoleFEE68E49', 'Arn' ] },
+                    ScalableDimension: 'dynamodb:table:ReadCapacityUnits',
+                    ServiceNamespace: 'dynamodb' } },
+              MyTableReadCapacityScalingPolicyCC18E396:
+               { Type: 'AWS::ApplicationAutoScaling::ScalingPolicy',
+                 Properties:
+                  { PolicyName: 'MyAwesomePolicyName',
+                    PolicyType: 'TargetTrackingScaling',
+                    ScalingTargetId: { Ref: 'MyTableReadCapacityScalableTarget72B0B3BF' },
+                    TargetTrackingScalingPolicyConfiguration:
+                     { PredefinedMetricSpecification: { PredefinedMetricType: 'DynamoDBReadCapacityUtilization' },
+                       ScaleInCooldown: 80,
+                       ScaleOutCooldown: 60,
+                       TargetValue: 75 } } } } });
+
+        test.done();
+    },
+
+    'when specifying Write Auto Scaling'(test: Test) {
+        const app = new TestApp();
+        const table = new Table(app.stack, 'MyTable', {
+            tableName: 'MyTable',
+            readCapacity: 42,
+            writeCapacity: 1337
+        });
+        table.addPartitionKey('partitionKey', KeyAttributeType.String);
+        table.addSortKey('sortKey', KeyAttributeType.Binary);
+        table.configureWriteAutoScaling({
```

Comment:
table.addWriteAutoScaling() would be a more consistent name

Reply:
Yep, makes sense. Let's get a second opinion.